nextjs-app/components/pages/main/WhatWeBring.tsx

```diff
@@ -0,0 +1,87 @@
+"use client";
+import Button from "@/components/common/Button";
+import Image from "next/image";
+import { useRouter } from "next/navigation";
+
+interface GoalCardProps {
+  imageSrc: string;
+  title: string;
+  subTitle: string;
+  description: string;
+}
+
+const GoalCard = (props: GoalCardProps) => {
+  const { imageSrc, title, subTitle, description } = props;
+  return (
+    <div className="flex flex-col items-center rounded-3 transition-all duration-300 border border-[#7E7059] bg-white z-10">
+      <div className="h-[30px]">
+        <Image
+          src={imageSrc}
+          alt={title}
+          width={120}
+          height={120}
+          className="-translate-y-1/2"
+        />
+      </div>
+      <div className="px-7 pt-11 pb-8 space-y-6">
+        <div className="space-y-2 text-[#7E7059]">
+          <h3 className="text-h4">{title}</h3>
+          <h6 className="text-h6">{subTitle}</h6>
+        </div>
+        <p className="text-3 leading-5">{description}</p>
```

Comment:
```suggestion
        <p className="text-body-lg text-neutral-10">{description}</p>
```

<img width="993" alt="image" src="https://github.com/user-attachments/assets/87d1977f-4bf3-40a0-a6d9-7db046f55e46" />